pages/docs/tutorials/generate-code.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Generate code 
+description: In this tutorial, you'll learn how to generate code from your AsyncAPI document.
+weight: 100
+---
+
+## Introduction
+
+In this tutorial, you'll learn how to generate code from your AsyncAPI document using the AsyncAPI generator tool.
+
+## Background context
+The [AsyncAPI Generator](https://github.com/asyncapi/generator) is a tool that you can use to generate whatever you want based on the AsyncAPI document. You can generate docs and code. It can be used as a library in Node.js application or through the [AsyncAPI CLI](https://github.com/asyncapi/cli).
+
+The generator tool supports a number of templates to generate code for a variety of different languages and protocols as the output. These templates help to specify what exactly must be generated, and in this tutorial, you use [Node.js template](https://github.com/asyncapi/nodejs-template).
+
+## Installation guide
+<Remember>
+
+It is important to have [Node.js](https://nodejs.org/en/download/) installed which will enable the installation of the required dependencies using <b>npm</b>.
+
+</Remember>
+
+- Install the generator tool to use it as a command-line (CLI):
+
+<CodeBlock language="bash">
+{`npm install -g @asyncapi/cli`}
+</CodeBlock>
+
+The command above helps you install the generator tool globally.
+
+- To install a specific version of the generator tool, pass the version during installation:
+
+<CodeBlock language="bash">
+{`npm install -g @asyncapi/cli@{version}`}
+</CodeBlock>
```

Comment:
```suggestion
```

this is a duplication of what you bring in with the CLI installation fragment document